app/models/race.rb

```diff
@@ -0,0 +1,142 @@
+    'american_indian_or_alaskan_native' => l10n('demographics.race.american_indian_or_alaskan_native'),
+    'other' => l10n('other')
+  }.freeze
+
+  # The undefined race options.
+  # @return [Array<String>] An array of undefined race options.
+  UNDEFINED_RACE_OPTIONS = %w[do_not_know refused].freeze
+
+  # The mapping of undefined race options to their human readable forms.
+  # @return [Hash] A hash mapping undefined race options to their human readable forms.
+  UNDEFINED_RACE_OPTIONS_MAPPING = {
+    'do_not_know' => l10n('do_not_know'),
+    'refused' => l10n('refused')
+  }.freeze
+
+  # The combined race options.
+  # @return [Array<String>] An array of all race options.
+  RACE_OPTIONS = (DEFINED_RACE_OPTIONS + UNDEFINED_RACE_OPTIONS).freeze
+
+  # The CMS reporting group kinds.
+  # @return [Array<String>] An array of CMS reporting group kinds.
+  CMS_REPORTING_GROUP_KINDS = %w[
+    white black_or_african_american american_indian_or_alaskan_native asian
+    native_hawaiian_or_other_pacific_islander multi_racial
+  ].freeze
+
+  # The mapping of CMS reporting group kinds to their human readable forms.
+  # @return [Hash] A hash mapping CMS reporting group kinds to their human readable forms.
+  CMS_REPORTING_GROUP_KINDS_MAPPING = {
+    'white' => 'White',
+    'black_or_african_american' => 'Black or African American',
+    'american_indian_or_alaskan_native' => 'American Indian or Alaskan Native',
+    'asian' => 'Asian',
+    'native_hawaiian_or_other_pacific_islander' => 'Native Hawaiian or Other Pacific Islander',
+    'multi_racial' => 'Multi Racial',
+    'unknown' => 'Unknown'
+  }.freeze
+
+  # The Races for each CMS reporting groups.
+  # @return [Array<String>] An array of Races for each CMS reporting groups.
+  RACES_FOR_CMS_GROUP_WHITE = %w[white].freeze
+  RACES_FOR_CMS_GROUP_BLACK_OR_AFRICAN_AMERICAN = %w[black_or_african_american].freeze
+  RACES_FOR_CMS_GROUP_AMERICAN_INDIAN_OR_ALASKAN_NATIVE = %w[american_indian_or_alaskan_native].freeze
+  RACES_FOR_CMS_GROUP_ASIAN = %w[asian_indian chinese filipino japanese korean vietnamese other_asian].freeze
+  RACES_FOR_CMS_GROUP_NATIVE_HAWAIIAN_OR_OTHER_PACIFIC_ISLANDER = %w[samoan native_hawaiian guamanian_or_chamorro other_pacific_islander].freeze
+  RACES_FOR_CMS_GROUP_UNKNOWN = %w[do_not_know refused].freeze
+
+  # The attestation kinds.
+  # @return [Array<String>] An array of attestation kinds.
+  # We could expand ATTESTATION_KINDS to include 'self_attested' 'broker_attested' and 'admin_attested'.
+  ATTESTATION_KINDS = %w[non_attested].freeze
+
+  # @!attribute [rw] attestation
+  #   @return [String, nil] The source of attestation for the ethnicity information.
+  #   This field is used to track the source of attestation.
+  #   It can be updated with the value 'non_attested' when we do not have ethnicity
+  #   information when migrating from the old data model to the new data model.
+  #   It is nil by default.
+  field :attestation, type: String
```

Comment:
same as in ethnicity - to me, name feels like it would/could hold an actual attested value. `attestation_kind` might help disambiguate.